src/containers/Booklist.js

```diff
@@ -0,0 +1,39 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BookList = (props) => {
+  const { books } = props;
+
+  const bookHtml = books.map((book) => {
+    const { id, title, category } = book;
+    return (<Book key={id} id={id} title={title} category={category} />);
```

Comment:
- [ ] [REQUIRED] It will be perfect if you pass directly `book object` as a prop here instead of passing id, title, category one by one